docs/changelog.md

```diff
@@ -75,6 +75,7 @@ As features stabilize some brief notes about them will accumulate here.
 * [wezterm cli rename-workspace](cli/cli/rename-workspace.md). #2787
 * [wezterm.mux.rename_workspace](config/lua/wezterm.mux/rename_workspace.md). #2787
 * [wezterm cli get-pane-direction](cli/cli/get-pane-direction.md)
+* [wezterm cli adjust-pane-size](cli/cli/adjust-pane-size.md) #3471
```

Comment:
This is in the previous stable release's entries! Move it up to the first `New` section.
Hmm, it looks like you're not rebased on `main`, so I don't see the current entries here in this diff.

https://github.com/wez/wezterm/blob/main/docs/changelog.md?plain=1#L27

Reply:
When you rebase, be sure to re-run `./ci/update-derived-files.sh && ./ci/build-docs.sh`

Reply:
Ah, `./ci/build-docs.sh` is failing for me, which I think is causing some confusion. I rebased on `main`, and as far as I can tell, this is the `New` section under `Continuous/Nightly` which seems like the right place?

Reply:
Okay I think I've hit everything now. I had the [1Password Shell Plugin](https://github.com/1Password/shell-plugins) for `gh` enabled and it wasn't working within the script.